src/main/java/com/aquarium/aquarium_backend/Security/AuthenticationRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.aquarium.aquarium_backend.Security;
+
+public class AuthenticationRequest {
+  private String username;
+  private String password;
+
+  public AuthenticationRequest(String username, String password) {
+    this.username = username;
+    this.password = password;
+  }
+
+  public AuthenticationRequest() {}
+
+  public String getUsername() {
+    return username;
+  }
+
+  public void setUsername(String username) {
+    this.username = username;
+  }
+
+  public String getPassword() {
+    return password;
+  }
+
+  public void setPassword(String password) {
+    this.password = password;
+  }
+}
```

Comment:
Consider using Dataclass here like: https://github.com/Aquatistic/Aquarium_backend/blob/29-add-sending-controll-data-to-effectors-from-backend/src/main/java/com/aquarium/aquarium_backend/Helpers/ControllStruct.java